feat : added alert, avatars and badges

ddivyansh18 opened the pull request:
Added alert, avatars and badges.
Looking for feedbacks and reviews.

ddivyansh18 committed:
feat : added alert, avatars and badges

Added alert, avatars and badges.

scjuly19 commented:
May I ask why have you defined html files for each component as I think the user will only want component styles right?

ddivyansh18 committed:
feat: added icon badges

Missed icon badges in previous commit, added them now

ddivyansh18 commented:
> May I ask why have you defined html files for each component as I think the user will only want component styles right?

I will use them later in documentation site.
Also it may help reviewers by letting them know how the components look visually.
Styles are mainly contained in css files, which will be further refactored later for documentation and bundling.

manish591 commented on Components/Alert/alert.css:
```diff
@@ -0,0 +1,66 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --theme-green: rgba(34, 197, 94, 0.4);
+  --theme-gray: rgba(15, 28, 39, 0.1);
+  --theme-yellow: rgba(253, 224, 71, 0.4);
+  --theme-red: rgba(239, 68, 68, 0.4);
+  --theme-blue: rgba(56, 189, 248, 0.4);
+  --theme-purple: rgba(99, 102, 241, 0.2);
+  --theme-black: rgb(16, 14, 9);
+}
```
I saw you are copying your CSS variables and reset code in every file. Instead of doing this, you can create a separate file where you can put this code. Then import that file wherever you need that code.

ddivyansh18 replied:
Moved common CSS to default.css, refactored the files accordingly

manish591 commented on Components/Alert/alert.css:
```diff
@@ -0,0 +1,66 @@
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin: 0.5rem 0;
```
## Avoid margins on your component
It will affect other elements when you use your component library to build other projects.
If you want to create a gap between your component then give their parent flex container a gap property.

ddivyansh18 replied:
Removed margins and used gap properties in the containers

ddivyansh18 committed:
fix : CSS related fixes

- Moved common CSS to default.css
- Removed margins and used gap property for layout

ddivyansh18 commented:
Hey @manish591 !
If everything looks fine, can you please approve the changes.
Thanks

manish591 reviewed (approved):
Now looks good
